lib/matplotlib/dviread.py

```diff
@@ -67,43 +69,12 @@ class Text(namedtuple('Text', 'x y font glyph width')):
     """
     A glyph in the dvi file.
 
-    The *x* and *y* attributes directly position the glyph.  The *font*,
-    *glyph*, and *width* attributes are kept public for back-compatibility,
```

Comment:
What about the *glyph* and *width* fields? Are they used?

Reply:
I restored their documentation.  There's not plan to deprecate them (they are useful anyways, and deprecating fields in a namedtuple is a bit annoying) so I removed the mention of backcompat.

Reply:
Not sure I see the docs back again?